docs/sources/setup-grafana/configure-security/configure-scim-provisioning/_index.md

```diff
@@ -0,0 +1,122 @@
+---
+aliases:
+  - ../../auth/scim/
+  - ../../enterprise/scim/
+description: Learn how to use SCIM provisioning to synchronize users and groups from your identity provider to Grafana.
+keywords:
+  - grafana
+  - scim
+  - provisioning
+  - user-management
+  - team-management
+labels:
+  products:
+    - cloud
+    - enterprise
+menuTitle: Configure SCIM provisioning
+title: Configure SCIM provisioning
+weight: 300
+---
+
+# Configure SCIM provisioning
+
+System for Cross-domain Identity Management (SCIM) is an open standard that allows automated user provisioning and management. With SCIM, you can automate the provisioning of users and groups from your identity provider to Grafana.
+
+> **Note:** Available in [Grafana Enterprise](../../../introduction/grafana-enterprise/) and [Grafana Cloud Advanced](/docs/grafana-cloud/).
+
+{{% admonition type="note" %}}
+This feature is behind the `enableSCIM` feature toggle.
+You can enable feature toggles through configuration file or environment variables. See configuration [docs](/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana/#feature_toggles) for details.
+{{% /admonition %}}
```

Comment:
- [The percent syntax for admonitions is discouraged](https://grafana.com/docs/writers-toolkit/review/lint-prose/rules/#grafanashortcodes)
- [Prefer "refer to" instead of "see"](https://grafana.com/docs/writers-toolkit/review/lint-prose/rules/#grafanareferto)
- [Write useful link text](https://grafana.com/docs/writers-toolkit/write/links/useful-link-text/)

```suggestion
{{< admonition type="note" >}}
This feature is behind the `enableSCIM` feature toggle.
You can enable feature toggles through configuration file or environment variables.

For more information, refer to the [feature toggles documentation](/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana/#feature_toggles).
{{< /admonition >}}
```